features/cli/oc_volume.feature

```diff
@@ -78,18 +78,17 @@ Feature: oc_volume.feature
     Then the step should succeed
 
     When I run the :get client command with:
-      | resource       | :false    |
-      | resource_name  | dc/mydc   |
-      | resource_name  | rc/mydc-1 |
-      | o              | yaml      |
+      | resource       | :false                                                   |
+      | resource_name  | dc/mydc                                                  |
+      | o             | jsonpath={.spec.template.spec.containers[*].volumeMounts} |
+    Then the step should succeed
+    And the output should contain 1 times:
+	    | name:secret |
+
+    When I run the :get client command with:
+      | resource       | :false                                                   |
```

Comment:
Why do we need this ?

Reply:
In the original version, we checked the output for dc and rc together, it's unstable, so I checked them separately. One for dc, one for rc.

Looks there are some format issues, fixed them.